robots/EDUCATOR/gyro.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+"""
+Move robot in a square path using the Gyro sensor.
+
+This script is a simple demonstration of turning using the Gyro sensor.
+"""
+
+from ev3dev.motor import MoveSteering
+from ev3dev.sensor.lego import GyroSensor
+
+
+class GyroSensorEnhanced(GyroSensor):
+    """Enhanced Gyro."""
+
+    def __init__(self):
+        """Create Gyro object if connected."""
+        super().__init__()
+        assert self.connected, 'Connect a Gyro to a sensor port'
+
+    def wait_until_changed_by(self, delta):
```

Comment:
@dwalton76 What are your thoughts on making this core API? We could do similar things for the ultrasonic sensor and others. Probably not an "initial release" task though.